keps/sig-windows/4885-windows-cpu-and-memory-affinity/README.md

```diff
@@ -0,0 +1,594 @@
+to help raise awareness of this possibility. If access from the CPUs different than the assigned Numa Node is undesirable then `single-numa-node` 
+and the CPU manager should be configured in the Topology Manager policy setting which would force Kubelet to only select a Numa node if it will have enough memory 
+and CPU's available.  In the future, in the case of workloads that span multiple Numa nodes, it may be desirable for Topology manager to have a new policy specific 
+for Windows. This would require a separate KEP to add a new policy.
+
+#### Kubelet memory management 
+
+Windows support for [kubelet's memory eviction](https://github.com/kubernetes/kubernetes/pull/122922) was enabled in 1.31 and would follow the same patterns
+as [Mechanism I](#mechanism-i-pod-eviction-by-kubelet).
+Windows does not have an OOM killer and so Mechanisms II and III are out of scope in the section 
+related to the [Kubernetes Node Memory Management](#kubernetes-nodes-memory-management-mechanisms-and-their-relation-to-the-memory-manager).
+
+### Windows Topology manager considerations
+
+Topology manager is already enabled on Windows in order to support the device manager.  Enabling the CPU and Memory manager as 
+hint providers will be behind a feature flag. The CPU manager and Memory Manager can independently be enabled or disabled to support cases where the features needs to be shut off.  
+
+### Test Plan
+
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+The testing plan is to enable basic tests in [Windows testing folder](https://github.com/kubernetes/kubernetes/tree/master/test/e2e/windows) in Alpha.  This will enable us to progress to a state we in Alpha that will allow our end users to test and give feedback in real world scenarios. 
+
+We we also work to enable e2e_node test suite to run on Windows and enable the applicable [CPU](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/container_manager_test.go)/[Memory](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/memory_manager_test.go)/[Topology](https://github.com/kubernetes/kubernetes/blob/master/test/e2e_node/topology_manager_test.go) Manager tests for Beta.  The goal will be to enable as many of those tests as possible while recognizing some may not be applicable to Windows.  Where we find gaps we will fill them with Windows specific tests.
+
+##### Prerequisite testing updates
+
+##### Unit tests
+
+- pkg/kubelet/cm/container_manager_windows.go
+- pkg/kubelet/cm/internal_container_lifecycle_windows.go
+- pkg/kubelet/winstats/cpu_topology_test.go
+
+##### Integration tests
+
+Integration tests do not run on Windows. Functionality will be covered by unit and e2e tests.
+
+##### e2e tests
+
+-  e2e_node will need to be enabled for windows to add coverage
```

Comment:
This seems to be a very important point. Are you okay with blocking this feature based on this?

Reply:
The goal would be to enable this for Beta.  I don't see a sustainable way for sig-windows to enable enough tests to ensure quality for these features without leveraging existing tests.  Duplicating all the test in e2e for windows would be a large undertaking as well

Reply:
Maybe it should be clear that you intend to add a e2e_node like test for this one. I read this as we are going to enable all of the tests as part of this feature..